DotNet/overrides.js

```diff
@@ -41,4 +41,5 @@ module.exports = {
     // Skipped test running on Selenium3 but not on Selenium4
     "should send dom on edge legacy": {skip: true},
     "Should return exception in TestResultsSummary": {skipEmit: true},
+    'should get dom pseudo elements when check window': {skipEmit: true},
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9515763)
```suggestion
    "should get dom pseudo elements when check window": {skipEmit: true},
```